terraform/cloudwatch.tf.json.template.py

```diff
@@ -367,6 +367,25 @@ def prod_qualified_resource_name(name: str) -> str:
                             },
                             'alarm_actions': ['${data.aws_sns_topic.monitoring.arn}'],
                             'ok_actions': ['${data.aws_sns_topic.monitoring.arn}'],
+                        },
+                        'waf_expensive_rate_blocked': {
+                            'alarm_name': config.qualified_resource_name('waf_expensive_rate_blocked'),
+                            'comparison_operator': 'GreaterThanThreshold',
+                            'threshold': 0,
+                            'datapoints_to_alarm': 1,
+                            'evaluation_periods': 1,
+                            'period': 5 * 60,
+                            'metric_name': 'BlockedRequests',
+                            'namespace': 'AWS/WAFV2',
+                            'statistic': 'Sum',
+                            'treat_missing_data': 'notBreaching',
+                            'dimensions': {
+                                'WebACL': '${aws_wafv2_web_acl.api_gateway.name}',
+                                'Region': config.region,
+                                'Rule': config.waf_expensive_rate_rule_name
+                            },
+                            'alarm_actions': ['${data.aws_sns_topic.monitoring.arn}'],
+                            'ok_actions': ['${data.aws_sns_topic.monitoring.arn}'],
                         }
                     }
```

Comment:
This block looks very similar to the one [preceding it](https://github.com/DataBiosphere/azul/pull/5869/commits/68550d82ab766bed41831024bd33be943b44d916#diff-e346abbea1127b30011301078954d7443ef3f3928ccdb47321a538b5fcc8e178R353-R366), consider de-duplicating by using a comprehension.

Reply:
Good call.